app/src/main/java/com/keylesspalace/tusky/components/notifications/NotificationsFragment.kt

```diff
@@ -434,6 +438,27 @@ class NotificationsFragment :
         }
     }
 
+    private fun removeDuplicateOlderEntries(positionStart: Int) {
+        val dataList = adapter.snapshot().items
+
+        Log.w(TAG, "found elements in adapter "+dataList.size)
```

Comment:
If you plan to keep these in the final code use `Log.d` please (or possibly `Log.v` if there are a lot of them per update).